ext/srt/gstsrtsink.c

```diff
@@ -312,6 +336,18 @@ gst_srt_sink_class_init (GstSRTSinkClass * klass)
       g_signal_new ("caller-rejected", G_TYPE_FROM_CLASS (klass),
       G_SIGNAL_RUN_LAST, 0, NULL, NULL, NULL, G_TYPE_NONE, 0);
 
+  /**
+   * GstSRTSink::caller-connecting:
+   * @gstsrtsink: the srtsink element that emitted this signal
+   *
+   * A caller's connection to srtsink.
```

Comment:
I'd explain the meaning of the callback return value here. From this simple description it isn't clear the signal listener can reject the connecting caller.

Reply:
Explained the meaning of the callback return value.